code/game/objects/items/tools/mining_tools.dm

```diff
@@ -192,12 +192,14 @@
 /obj/item/tool/pickaxe/plasmacutter/proc/calc_delay(mob/user)
 	var/final_delay = PLASMACUTTER_CUT_DELAY
 	if (istype(user))
-		if(user.mind && user.mind.cm_skills && user.mind.cm_skills.engineer < SKILL_ENGINEER_ENGI)
-			user.visible_message("<span class='notice'>[user] fumbles around figuring out how to use [src].</span>",
-			"<span class='notice'>You fumble around figuring out how to use [src].</span>")
-			final_delay *= max(1, 4 + (user.mind.cm_skills.engineer * -1)) //Takes twice to four times as long depending on your skill.
-		else
-			final_delay -= min(PLASMACUTTER_CUT_DELAY,(user.mind.cm_skills.engineer - 3)*5)
+		if(user.mind && user.mind.cm_skills)
+			var/engi_skill = user.mind.cm_skills.engineer
```

Comment:
Why did you make this a var? You can as well do simply `if(user.mind.cm_skills.engineer < SKILL_ENGINEER_ENGI)` since you are not modifying it or anything.

Reply:
Because it's referenced several times and it looks better/cleaner this way.